### PR TITLE
23 fix field names

### PR DESCRIPTION
**Pull request by Ozrei**

### CODE REVIEW

**Comment by enki211** [comment]
It looks like you need to fix the conflicts with master, as well as make a migration because you changed the models.

File: keyform/views.py [hunk 1]
```diff
@@ -37,6 +38,15 @@ def form_valid(self, form):
             return self.form_invalid(form)
 
     def get_form(self, form_class=None):
-        form = CreateForm(**self.get_form_kwargs())
+        form = CreateForm(instance=Request(staff=get_user_model().objects.all()[0]), **self.get_form_kwargs())
         form.request_formset = RequestFormSet(**self.get_form_kwargs())
         return form
+
+    def get_form_kwaargs(self):
```
Comment (shaneallgeier) [inline keyform/views.py]: `get_form_kwaargs()` -> `get_form_kwargs()`
Also figure out if this method is even necessary

Comment (Ozrei) [inline keyform/views.py]: Turns out it's not

File: keyform/views.py [hunk 2]
```diff
@@ -37,6 +38,15 @@ def form_valid(self, form):
             return self.form_invalid(form)
 
     def get_form(self, form_class=None):
-        form = CreateForm(**self.get_form_kwargs())
+        form = CreateForm(instance=Request(staff=get_user_model().objects.all()[0]), **self.get_form_kwargs())
```
Comment (shaneallgeier) [inline keyform/views.py]: This code is simply picking the first user instead of the currently logged in user.

**Comment by shaneallgeier** [comment]
[request.html, line 66](https://github.com/mostateresnet/keyformproject/blob/f44e998725cbb7b8c749465e92eed6dd50fc987e/keyform/templates/keyform/request.html#L66) still refers to the old, incorrect spelling of `amt_received`.

File: keyform/models.py [hunk 3]
```diff
@@ -40,13 +40,13 @@ class Request(models.Model):
...
-    amt_recieved = models.DecimalField(max_digits=7, decimal_places=2, default=0, blank=True, validators=[MinValueValidator(Decimal('0.00'))])
+    amt_received = models.DecimalField(max_digits=7, decimal_places=2, default=0, blank=True, verbose_name='Amount received', validators=[MinValueValidator(Decimal('0.00'))])
     payment_method = models.CharField(max_length=2, choices=PAYMENT_TYPES, null=True, blank=True)
     charge_amount = models.DecimalField(max_digits=7, decimal_places=2, validators=[MinValueValidator(Decimal('0.00'))])
     staff = models.ForeignKey(settings.AUTH_USER_MODEL)
-    bpn = models.CharField(max_length=9, validators=[bpn_validator])
+    bpn = models.CharField(max_length=9, verbose_name='M-Number', validators=[bpn_validator])
     created_timestamp = models.DateTimeField(default=now, blank=True)
-    charged_on_rcr = models.BooleanField(default=False)
+    charged_on_rcr = models.BooleanField(default=False, verbose_name='Charged on RCR')
```
Comment (shaneallgeier) [inline keyform/models.py]: Translate pls

File: keyform/models.py [hunk 4]
```diff
@@ -40,13 +40,13 @@ class Request(models.Model):
     building = models.ForeignKey(Building)
     student_name = models.CharField(max_length=128, blank=True)
     reason_for_request = models.CharField(max_length=2, choices=REQUEST_TYPES)
-    amt_recieved = models.DecimalField(max_digits=7, decimal_places=2, default=0, blank=True, validators=[MinValueValidator(Decimal('0.00'))])
+    amt_received = models.DecimalField(max_digits=7, decimal_places=2, default=0, blank=True, verbose_name='Amount received', validators=[MinValueValidator(Decimal('0.00'))])
     payment_method = models.CharField(max_length=2, choices=PAYMENT_TYPES, null=True, blank=True)
     charge_amount = models.DecimalField(max_digits=7, decimal_places=2, validators=[MinValueValidator(Decimal('0.00'))])
     staff = models.ForeignKey(settings.AUTH_USER_MODEL)
-    bpn = models.CharField(max_length=9, validators=[bpn_validator])
+    bpn = models.CharField(max_length=9, verbose_name='M-Number', validators=[bpn_validator])
```
Comment (shaneallgeier) [inline keyform/models.py]: Translate pls

File: keyform/models.py [hunk 5]
```diff
@@ -40,13 +40,13 @@ class Request(models.Model):
     building = models.ForeignKey(Building)
     student_name = models.CharField(max_length=128, blank=True)
     reason_for_request = models.CharField(max_length=2, choices=REQUEST_TYPES)
-    amt_recieved = models.DecimalField(max_digits=7, decimal_places=2, default=0, blank=True, validators=[MinValueValidator(Decimal('0.00'))])
+    amt_received = models.DecimalField(max_digits=7, decimal_places=2, default=0, blank=True, verbose_name='Amount received', validators=[MinValueValidator(Decimal('0.00'))])
```
Comment (shaneallgeier) [inline keyform/models.py]: Translate pls

File: keyform/views.py [hunk 6]
```diff
@@ -7,6 +7,7 @@
 from keyform.models import Request
 from django.urls import reverse_lazy
 from django.http import HttpResponseRedirect
+from django.contrib.auth import get_user_model
```
Comment (shaneallgeier) [inline keyform/views.py]: Unnecessary import

File: keyform/models.py [hunk 7]
```diff
@@ -40,13 +40,13 @@ class Request(models.Model):
...
-    amt_received = models.DecimalField(max_digits=7, decimal_places=2, default=0, blank=True, verbose_name='Amount received', validators=[MinValueValidator(Decimal('0.00'))])
+    amt_received = models.DecimalField(max_digits=7, decimal_places=2, default=0, blank=True, verbose_name= _('Amount received'), validators=[MinValueValidator(Decimal('0.00'))])
     payment_method = models.CharField(max_length=2, choices=PAYMENT_TYPES, null=True, blank=True)
     charge_amount = models.DecimalField(max_digits=7, decimal_places=2, validators=[MinValueValidator(Decimal('0.00'))])
     staff = models.ForeignKey(settings.AUTH_USER_MODEL)
-    bpn = models.CharField(max_length=9, verbose_name='M-Number', validators=[bpn_validator])
+    bpn = models.CharField(max_length=9, verbose_name=_('M-Number'), validators=[bpn_validator])
     created_timestamp = models.DateTimeField(default=now, blank=True)
-    charged_on_rcr = models.BooleanField(default=False, verbose_name='Charged on RCR')
+    charged_on_rcr = models.BooleanField(default=False, verbose_name=_('Charged on RCR')
```
Comment (shaneallgeier) [inline keyform/models.py]: Syntax error

File: keyform/templates/keyform/login.html [hunk 8]
```diff
@@ -0,0 +1,29 @@
+{% extends "keyform/base.html" %}
```
Comment (enki211) [inline keyform/templates/keyform/login.html]: Why is the login template part of this pull request?

**Comment by enki211** [comment]
Although you did not add this since you already touch this file can you remove [this](https://github.com/mostateresnet/keyformproject/blob/936b279c060c40fd47034a04a3bef8691f326b19/keyform/templates/keyform/request.html#L13) line?

**Pull request merged**